src/VisualStudio/Core/Def/Implementation/Diagnostics/VisualStudioDiagnosticAnalyzerService.cs

```diff
@@ -179,7 +179,7 @@ private void OnRunCodeAnalysisForSelectedProject(object sender, EventArgs args)
         public void RunAnalyzers(IVsHierarchy? hierarchy)
         {
             var project = GetProject(hierarchy);
-            var solution = _workspace.CurrentSolution;
+            Solution solution = _workspace.CurrentSolution;
```

Comment:
❔ Was this just a test or was it needed for some reason?

Reply:
Explained in OP: Filed issue #40821 to track the remaining issue with analysis of local functions which affected bootstrapping for this PR.

The code above behaves like `Solution? solution = ...;` which hits a problem with analysis of local functions. Most such problems were fixed by Andy yesterday, but one or two remained, which I worked around to unblock this PR (because it's not a new problem).

---
In reply to: [364502291](https://github.com/dotnet/roslyn/pull/40755#discussion_r364502291) [](ancestors = 364502291)